front/lib/data_sources.ts

```diff
@@ -1,36 +1,4 @@
-import { CoreAPIDocument, DataSourceType } from "@dust-tt/types";
-
-export function getProviderLogoPathForDataSource(
```

Comment:
Reason for deleting this code:
- The search page is not used anymore.
- We are not supposed to use these logo anymore, we are supposed to use Icons instead